src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -96,131 +96,135 @@
             }
         };
 
+        var successStreak = 4;
+        var pendingRequests = 0;
         var renderAjaxRequests = function() {
-            var requestCounter = document.querySelectorAll('.sf-toolbar-ajax-requests');
-            if (!requestCounter.length) {
+            var requestCounter = document.querySelectorAll('.sf-toolbar-ajax-requests')[0];
+            if (!requestCounter) {
                 return;
             }
+            requestCounter.textContent = requestStack.length;
```

Comment:
this DOM updates should be done *after* the retrieval of other needed nodes, to group read and writes